ModKit/UI/UI+Elements.cs

```diff
@@ -4,12 +4,12 @@
 namespace ModKit {
     public static partial class UI {
 
-        public const string ChecklyphOn = "<color=green><b>✔</b></color>";
```

Comment:
please don't replace these values.  Please provide and alternate set of glyphs and put it behind a settings toggle.

Reply:
From
https://github.com/cabarius/ToyBox/issues/499
https://github.com/cabarius/ToyBox/issues/483

@cabarius could you point us in the right direction for that? Appologies, its not super clear where to do so.

Reply:
We could create annother class, is that the intent? CharcodeUI ?